_rules/text-contrast-afw4f7.md

```diff
@@ -183,6 +183,36 @@ This text is using the default user agent text color and background color. By de
 <div role="button">My button!</div>
 ```
 
+#### Passed Example 12
+
+This 18pt bold black Japanese text has a contrast ratio of 3.6:1 on the gray background.
+
+```html
+<p style="color: #000; font-size:18pt; font-weight:700; background: #666;" lang="ja">
+	中国語、日本語、及び韓国語
+</p>
+```
+
+#### Passed Example 13
+
+This 22pt black Chinese text has a contrast ratio of 3.6:1 on the gray background.
+
+```html
+<p style="color: #000; font-size:22pt; font-weight:700; background: #666;" lang="ja">
+	内容无障碍指南
+</p>
+```
+
+#### Passed Example 14
+
+This 18pt black Japanese text has a contrast ratio of 3.6:1 on the gray background. Since the characters composing the sentence are not included within the Unicode standard range of 4E00–9FFF, this is [large scale text][] and passes the rule.
+
+```html
+<p style="color: #000; font-size:18pt; background: #666;" lang="ja">
+	「ロック・アンド・ロール」
+</p>
+```
+
```

Comment:
#### Passed Example 15

This 16pt bold black Traditional Chinese text has a contrast ratio of 3.6:1 on the gray background.

```html
<p style="color: #000; font-size:18pt; font-weight:700; background: #666;" lang="zh-TW">
	內容可及性規範
</p>
```